web/docs/deployment/deployment-methods/_addExternalAuthEnvVarsReminder.md

```diff
@@ -1,3 +1,3 @@
 :::tip Using an external auth method?
-If your app is using an external authentication method(s) supported by Wasp (such as [Google](../../auth/social-auth/google#4-adding-environment-variables) or [GitHub](../../auth/social-auth/github#4-adding-environment-variables)), make sure to additionally set the necessary environment variables specifically required by these method(s).
+If your app is using an external authentication method(s) supported by Wasp (such as [Google](/auth/social-auth/google.md#4-adding-environment-variables) or [GitHub](/auth/social-auth/github.md#4-adding-environment-variables)), make sure to additionally set the necessary environment variables specifically required by these method(s).
```

Comment:
I've been using this for some time now and it works great 👍 , it's explained in Writing Guide https://github.com/wasp-lang/wasp/blob/main/web/docs/writingguide.md#linking-to-pages-in-the-docs

Reply:
That is suprising to me because it sounds like it goes against theri docs. Just to confirm very explicitly: you can confirm this works if we release new versioned docs and these become numbered as let's say 0.15? Will these links, when clicked from 0.15 version of docs, correctly take you to the 0.15 version of these files, or instead incorrectly to the most current docs?